digital_comms/ccam.py

```diff
@@ -307,10 +447,39 @@ def pairwise(iterable):
 def lookup_clutter_geotype(clutter_lookup, population_density):
     """Return geotype based on population density
 
-    Params:
-    ======
-    clutter_lookup : list of (population_density_upper_bound, geotype) tuples
-        sorted by population_density_upper_bound ascending
+    Parameters
+    ----------
+    clutter_lookup: list of tuple
+        Lookup table that represents geographical types and their population density
+        sorted by population_density_upper_bound ascending.
+
+        * 0: :obj:`int`
+            Population density in TODO
```

Comment:
population density should be in persons per square kilometer

Reply:
Done